Dockerfile.prod

```diff
@@ -25,31 +29,29 @@ RUN yarn list
 # Building TypeScript files
 RUN yarn run build:release
 
-# https://stackoverflow.com/questions/57108751/docker-build-failed-when-copying-in-multi-step-build
-RUN chown root:root -R /app
+# Change back to root user to change ownership
+USER root
+RUN chown root:root -R /opt/service
 
-FROM node:14.21-alpine3.16
+FROM docker.atl-paas.net/sox/micros-node-14:1.0.5
 
-# adding to solve vuln
-RUN apk add --update --upgrade busybox
-RUN apk add --update --upgrade openssl
-RUN apk add --update --upgrade zlib
+# root user for installs
+USER root
 
 # For debugging curl command
-RUN apk add curl
+RUN apt-get install -y curl
 
 # For coredumps
-RUN apk add gdb
-RUN apk add bash
+RUN apt-get install -y gdb
 
-USER node
-COPY --chown=node:node --from=build /app /app
+# non-root user
+USER appuser
```

Comment:
appuser is a predefined user on the image, node user does not exist and appuser is same-same-but-dfferent